content/docs/3.0/tasks/best-practices/pd-scheduling.md

```diff
@@ -0,0 +1,268 @@
+---
+title: PD Scheduling
+description: Learn best practice and strategy for PD scheduling.
+menu:
+    docs:
+        parent: Best Practices
+        weight: 1
+---
+
+This document details the principles and strategies of PD scheduling through common scenarios to facilitate your application. This document assumes that you have a basic understanding of TiDB, TiKV and PD with the following core concepts:
```

Comment:
Generally tikv.org documentation doesn't discuss TiDB, documentation discussing TiDB should be on PingCAP's site. :)

Reply:
Got it! Thanks!